src/main/java/com/mhrglobal/domain/Director.java

```diff
@@ -0,0 +1,25 @@
+package com.mhrglobal.domain;
+
+
+public class Director extends Employee {
```

Comment:
Director implementation is capable of calculating the total salary. It can not have extra hours.